charts/semaphore/templates/postgresql/persistent-volume-claim.yaml

```diff
@@ -9,10 +9,10 @@ metadata:
   name: {{ include "semaphore.database.volume.name" . }}
 spec:
   accessModes:
-    - ReadWriteOnce
+    {{ toYaml .Values.database.persistence.accessModes | indent 4}}
   resources:
     requests:
-      storage: {{ include "semaphore.database.volume.size" . }}
+      storage: "{{ default "10Gi" .Values.database.persistence.volumeSize }}"
```

Comment:
Let's have a default in values.yaml, and not here.

Reply:
values.yaml have value, this was meant as fallback value

removed it